src/out_hoi4/portraits/out_portraits.cpp

```diff
@@ -0,0 +1,27 @@
+#include "src/out_hoi4/portraits/out_portraits.h"
```

Comment:
Add a test for this file, verifying that converter_portraits.txt is created and multiple countries written